turbinia/workers/psort.py

```diff
@@ -24,7 +24,46 @@
 
 
 class PsortTask(TurbiniaTask):
-  """Task to run Psort to generate CSV output from plaso storage files."""
+  """Task to run Psort (Plaso toolset)
+
+    Args:
+      base_command (str): Command to invoke log2timeline (e.g. log2timeline.py)
```

Comment:
I think this is the docstring for `build_plaso_command` and not the object `PsortTask`.